dev.edfloreshz.Done.Devel.json

```diff
@@ -0,0 +1,89 @@
+{
+    "app-id" : "dev.edfloreshz.Done.Devel",
```

Comment:
In general Flathub is not for development snapshots,

Reply:
Oh okay, sorry about that, I saw some apps on the repo ending in `.Devel` and wanted to upload my app as well.

Reply:
he very few are EXCEPTIONS.

Reply:
In what cases are these exceptions made?

Reply:
We accepted a single `Devel` app. Flathub is generally meant for stable releases, developers can also use `beta` branch for testing.